src/testing/flow_solver.cpp

```diff
@@ -96,6 +96,12 @@ int FlowSolver<dim,nstate>::run_test() const
         flow_solver_case->steady_state_postprocessing(dg);
     }
     pcout << "done." << std::endl;
+}
+
+template <int dim, int nstate>
+int FlowSolver<dim,nstate>::run_test() const
```

Comment:
Is there a reason for creating run_flow_solver instead of just having it in run_test?

Reply:
Yeah, since run test returns an `int`, having a `void` function lets us avoid that

Reply:
Its a bit confusing to have an extra function just to avoid returning a value. If you just use the run_test function and don't use the return value, does the compiler complain? If it does you could try static_cast<void>(run_test()).

Reply:
I think eventually `FlowSolver` should be given its own directory and no longer be in /src/testing, since its not really a test. In that case, `run_flow_solver()` does make more sense. But for now, to run it outside of the class, instead of `int success = flow_solver.run_test()`, you could try `static_cast<void>(run_test());`, I think it'll eliminate the need to pass the return value to a variable (not sure I haven't tried it). Otherwise, I guess you can just leave it as it is, and we can change it someday if we pull `FlowSolver` out of `/src/testing`.

Reply:
Agreeing with Donovan here. The flow solver would be moved out at some point to its own directory and it would return a more significant enum regarding convergence. For now the scope of this PR is to add the integration test. Simply open an issue to move the flow solver, reference this conversation, and resolve this conversation.

Regarding the suppression of warning error: casting it C-style as `(void)run_test();` is what I have been doing and used more commonly for unused variables.

Reply:
Addressed in most recent commit